pages/glossary/content-that-becomes-visible.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Content that becomes visible
+key: content-that-becomes-visible
+unambiguous: true
+objective: true
+input_aspects:
+  - CSS styling
+  - DOM tree
+---
+
+The _content that becomes visible_ after an [event][] [firing][] is any element for which all of the following is true:
+
+- 500 milliseconds after the [event][] [firing][] the element has [visible text content][]; and
```

Comment:
We need to capture an assumption that content on hover will appear within 500ms.